vm_video_configuration_field.md: update beta and GA feature stage

dasionov opened the pull request:
### VEP Metadata

**Tracking issue**: <!-- For example, https://github.com/kubevirt/enhancements/pull/2 -->
**SIG label**:  /sig compute

Relates to #35

### What this PR does
Added Beta stage updates, tests and documentation which needs to be done.

### Special notes for your reviewer

enp0s3 commented on veps/sig-compute/vm_video_configuration_field.md:
```diff
@@ -148,4 +148,19 @@ We are targeting this feature to be introduced in **Alpha**.
    - The `video` field will be **optional**.
    - Users can opt into the new functionality without affecting existing behavior.
 
+- During Beta:
```
@dasionov From capacity perspective are you sure that you will make it with all the testing for 1.7?

xpivarc reviewed (commented):
/approve
/hold

@dasionov Please address @enp0s3 comments. I think you will need to call out for short exception, please respond to @vladikr mail thread.

dasionov commented on veps/sig-compute/vm_video_configuration_field.md:
```diff
@@ -148,4 +148,19 @@ We are targeting this feature to be introduced in **Alpha**.
+  - Document supported video models per architecture, including multi-head
+    support where applicable.
+
+  - Test that Windows guests (including Windows 11 on ARM) can take advantage
```
I was basing this on this comment from @ksimon1 [here](https://github.com/kubevirt/enhancements/pull/36#issuecomment-2841482743)

dasionov commented:
[Changes](https://github.com/kubevirt/enhancements/compare/2406aa308c784c3a2a26e8a9258ca6b83e9046cc..6ae8dcbdf8104970656508f26fe98e175983d4e9) 
- Added a support matrix that documents supported video models per architecture, firmware type (BIOS/EFI), and guest OS type.

- Added a GA section following the standard enhancement template, covering API stability, defaults, validation, backward compatibility, and testing.

enp0s3 commented:
@dasionov Thank you!
/unhold
